### ISSUE TITLE
Highlighted story issues

### ISSUE THREAD

**Issue by matthewberryman** [AcrossTheCloud/TBA21-client#354]
* Title should be in Amare (please check).
* Make the width of the title element a bit smaller (I am happy to do this one if you can point me at where in the css to save me a bit of time)
* Excerpt text needs to fit inside a box the same width as the title (see screenshot below).
* Excerpt text should be centre aligned within that box.
* Don't display excerpt text on mobile (no hover view) / narrow desktop (unlikely)
<img width="271" alt="Screen Shot 2021-09-08 at 1 45 21 pm" src="https://user-images.githubusercontent.com/2288238/132443162-40c5de17-7df2-4424-885f-ca2a539ed3bc.png">

**Comment by matthewberryman** [comment]
Missing ellipse when excerpt is truncated

**Comment by philipyoungg** [comment]
Ellipsis only works with one line. Multiline ellipsis need weird tricks that don't work on all browser https://css-tricks.com/multi-line-truncation-with-pure-css/

Perhaps just adding `:after {content: "..."}` would be enough?

**Comment by matthewberryman** [comment]
sounds like an idea worth trying

**Comment by philipyoungg** [comment]
Nah, it can't be done. I don't think it's worth it to pursue this, unless we limit the story excerpt to first n characters and add '...' at the end?

**Comment by matthewberryman** [comment]
Yes (but with n words, not characters)

**Comment by philipyoungg** [comment]
![CleanShot 2021-09-08 at 18 57 44@2x](https://user-images.githubusercontent.com/11477718/132505217-c0dae35f-afb2-41e4-8ac5-8600e34e004b.jpg)
